Header header
bool armed
uint8 ctrl_mode
uint8 nav_state
uint8 nav_goal
uint8 nav_solution
float32 battery_voltage


